### DISCUSSION TITLE
Tesla Java API / Implementation NoMFA "unsupported SSO app"

### DISCUSSION THREAD

**Discussion by PatrickSchmidtSE** [timdorr/tesla-api#290] [Q&A]
The NoMFAWAY leads to an "unsupported SSO app". I converted the code to my .NET Test app and ultimatly end up in the same error :/ 
https://github.com/tbee/teslaApi/blob/main/src/main/java/org/tbee/tesla/TeslaNoMFALogic.java

Is there any solution to this, or do i need to implement it like here? 
https://github.com/tbee/teslaApi/blob/main/src/main/java/org/tbee/tesla/TeslaMFALogic.java

**Comment by dacastro4** [comment]
It looks like he updated the methods. I'm going to test them out

**Reply by dacastro4** [reply]
Still getting "unsupported SSO app

**Reply by dacastro4** [reply]
After a couple of hours working on it. I made it work. I had to change some values and add the same headers that the PHP and JS examples are sending.

**Reply by vvanengers** [reply]
I am trying to use the library. How did you solve the unsupported SSO app problem?
